### PR TITLE
CI updates

### PR DESCRIPTION
**Pull request by stone-z**
- Add dependabot config
- Update architect orb
- fix linting issues - remove unused code, catch error return, simplify

### CODE REVIEW

**Comment by stone-z** [comment]
Will update the required checks for this repo once merged

**Comment by stone-z** [comment]
Running on geckon

File: helm/kvm-operator/ci/default-values.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
```
Comment (MarcelMue) [inline helm/kvm-operator/ci/default-values.yaml:1]: Why is this needed?

Comment (stone-z) [reply helm/kvm-operator/ci/default-values.yaml:1]: The file is needed to keep the new helm chart linting happy

The line might not be necessary, I just based this off of what's in use elsewhere
